### PR TITLE
Docs(*)ADD File README.md

### PR DESCRIPTION
**Pull request by jamessantos10**
Esse é ReadME referente ao respositório mongo-collector-metrics.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
...
+
+## Install 
+
+We run Deploy in production of the mongo-collector-metrics application, as follows:
+
+`kubectl apply -f https://github.com/GrupoZapVivaReal/vivareal-graylog/tree/master/deploy-k8s/40-secrets/secret-mongo-collector-metrics.yaml`
+
+`kubectl apply -f https://github.com/GrupoZapVivaReal/vivareal-graylog/tree/master/deploy-k8s/90-collector-mongo `
+
+After that, we access the link http://prometheus.private.prod.vivareal.io and run the following query `mongodb_mongod_connections`, then see the result below as evidence that the logs are being sent to prometheus:
```
Comment (souzabrunoftw) [inline README.md]: The `PROD` URL shouldn't be fixed, gives the impression that `QA` doesn't exist.
Maybe use a placeholder like `http://prometheus.private.<env>.vivareal.io`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,32 @@
...
+
+![captura de tela de 2018-05-17 14-01-31](https://user-images.githubusercontent.com/35613398/40192456-d553e1a4-59da-11e8-83ec-b4e148f895f1.png)
```
Comment (souzabrunoftw) [inline README.md]: Hide Chrome tabs :smile: and maybe store the image in the directory, in an assets folder

File: README.md [hunk 3]
```diff
@@ -1,2 +1,32 @@
...
+
+## Usage
+
+Access Prometheus  QA or Prod interface and execute this command **mongodb_mongod** and select query you prefer.
+![captura de tela de 2018-06-06 15-48-30](https://user-images.githubusercontent.com/35613398/41058803-1d3f875e-69a1-11e8-8ab0-1c94c641c37f.png)
```
Comment (souzabrunoftw) [inline README.md]: Hide Chrome tabs :smile: and maybe store the image in the directory, in an assets folder

File: README.md [hunk 4]
```diff
@@ -1,2 +1,32 @@
...
+## Features
+
+- MongoDB Server Status metrics (cursors, operations, indexes, storage, etc)
+- MongoDB Replica Set metrics (members, ping, replication lag, etc)
+- MongoDB Replication Oplog metrics (size, length in time, etc)
+- MongoDB Sharding metrics (shards, chunks, db/collections, balancer operations)
+- MongoDB RocksDB storage-engine metrics (levels, compactions, cache usage, i/o rates, etc)
+- MongoDB WiredTiger storage-engine metrics (cache, blockmanger, tickets, etc)
+
+## Install
```
Comment (souzabrunoftw) [inline README.md]: Install should be **How to Deploy**

**Review by souzabrunoftw — CHANGES_REQUESTED** [review]
Few points I think you should consider. Also, congratz on the great doc!!

File: README.md [hunk 5]
```diff
@@ -1,2 +1,33 @@
 # mongo-collector-metrics
 this repository was created for the mongodb metrics collector, this app sends metrics to the prometheus
```
Comment (souzabrunoftw) [inline README.md]: `this` should be uppercase.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,33 @@
...
+
+We run Deploy in production of the mongo-collector-metrics application, as follows:
+
+`kubectl apply -f https://github.com/GrupoZapVivaReal/vivareal-graylog/tree/master/deploy-k8s/40-secrets/secret-mongo-collector-metrics.yaml`
+
+`kubectl apply -f https://github.com/GrupoZapVivaReal/vivareal-graylog/tree/master/deploy-k8s/90-collector-mongo `
+
+After that, we access the link http://prometheus.private.<env>.vivareal.io and run the following query `mongodb_mongod_connections`, then see the result below as evidence that the logs are being sent to prometheus:
+
+![image](https://user-images.githubusercontent.com/28811010/41117258-e661e35e-6a62-11e8-9032-1b431a910a90.png)
```
Comment (souzabrunoftw) [inline README.md]: This screenshot is better, but you could cut the blank part in the bottom.

File: README.md [hunk 7]
```diff
@@ -1,2 +1,33 @@
...
+
+## Usage
+
+Access Prometheus  QA or Prod interface and execute this command **mongodb_mongod** and select query you prefer.
```
Comment (souzabrunoftw) [inline README.md]: Too many blank spaces between `Prometheus` and `QA`

**Review by souzabrunoftw — CHANGES_REQUESTED** [review]
Just a few more points I've noticed. Sorry for being annoying ;)

**Review by souzabrunoftw — APPROVED** [review]

**Review by williamnaltchadjian — APPROVED** [review]
LGTM

**Pull request merged**